src/main/java/mate/academy/spring/boot/service/ShoppingCartService.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.boot.service;
+
+import mate.academy.spring.boot.dto.cartitem.CartItemQuantityRequestDto;
+import mate.academy.spring.boot.dto.cartitem.CreateCartItemRequestDto;
+import mate.academy.spring.boot.dto.shoppingcart.ShoppingCartDto;
+import mate.academy.spring.boot.model.ShoppingCart;
+import mate.academy.spring.boot.model.User;
+
+public interface ShoppingCartService {
+    ShoppingCart getShopCart(Long userId);
```

Comment:
You should have these methods: Don't add redundant or unused methods
```suggestion
    void createShoppingCart(User user);

    ShoppingCartDto getUsersCart(Long userId);

    ShoppingCartDto addBookToCart(Long userId, CartItemRequestDto item);

    ShoppingCartDto updateBookQuantity(Long userId, Long cartItemId, UpdateCartItemDto itemDto);

    void deleteBookFromCart(Long userId, Long itemId);
```